src/main/java/gameboy/gamepack/racinggame/view/ResultView.java

```diff
@@ -0,0 +1,19 @@
+package gameboy.gamepack.racinggame.view;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ResultView {
+
+    public static void display(List<Integer> carsPosition) {
+        System.out.println(getRaceResult(carsPosition));
+    }
+
+    public static String getRaceResult(List<Integer> carsPosition) {
+        return carsPosition.stream().map(ResultView::createWheelMark).collect(Collectors.joining("\n","", "\n"));
```

Comment:
```suggestion
        return carsPosition.stream()
        .map(ResultView::createWheelMark)
        .collect(Collectors.joining("\n","", "\n"));
```

코드가 가로로 늘어지면 보기 힘들 수 있습니다.
stream 메서드를 사용할 때는 짧게 짧게 끊어서 읽는 게 더 보기 좋습니다.